pom.xml

```diff
@@ -1500,6 +1500,8 @@ under the License.
                         <exclude>flink-formats/flink-avro/src/test/resources/flink_11-kryo_registrations</exclude>
 						<exclude>flink-scala/src/test/resources/flink_11-kryo_registrations</exclude>
 						<exclude>flink-core/src/test/resources/kryo-serializer-config-snapshot-v1</exclude>
+						<exclude>flink-core/src/test/resources/abstractID-with-toString-field</exclude>
+						<exclude>flink-core/src/test/resources/abstractID-with-toString-field-set</exclude>
```

Comment:
Why add these two items?  Will the test fail without this change?

Reply:
When the rat-plugin scans the license at compile time, it will scan these files and cause errors, but these files do not require the license

Reply:
These files have been added for a long time, and the master build is always passing. I don't understand why it failed in your case. Could you try to remove the changes and see if the tests fail?

Reply:
![image](https://user-images.githubusercontent.com/58294989/218654777-9b002e1f-973b-48fc-9fad-0b6813afd094.png)
When I remove these exceptions locally, I get an error when I compile

Reply:
![image](https://user-images.githubusercontent.com/58294989/218655012-a37c8056-68cd-4aab-aa6f-aa9382d875be.png)

Reply:
Which maven version are you using? Could you try v3.2.5 ?

Reply:
My maven version is v3.6.3, I tried v3.2.5 but it didn't work

Reply:
public static boolean isBinary(InputStream in) {
        try {
            byte[] taste = new byte[200];
            int bytesRead = in.read(taste);
            if (bytesRead > 0) {
                ByteBuffer bytes = ByteBuffer.wrap(taste, 0, bytesRead);
                CharBuffer chars = CharBuffer.allocate(2 * bytesRead);
                CharsetDecoder cd = CHARSET_FROM_FILE_ENCODING_OR_UTF8.newDecoder()
                        .onMalformedInput(CodingErrorAction.REPORT)
                        .onUnmappableCharacter(CodingErrorAction.REPORT);
                while (bytes.remaining() > 0) {
                    CoderResult res = cd.decode(bytes, chars, true);
                    if (res.isMalformed() || res.isUnmappable()) {
                        return true;
                    } else if (res.isOverflow()) {
                        chars.limit(chars.position());
                        chars.rewind();
                        int c = chars.capacity() * 2;
                        CharBuffer on = CharBuffer.allocate(c);
                        on.put(chars);
                        chars = on;
                    }
                }
                chars.limit(chars.position());
                chars.rewind();
                return isBinary(chars);
            }
        } catch (IOException e) {
            // SWALLOW 
        }
        return false;
    }

    static Charset getFileEncodingOrUTF8AsFallback() {
        try {
            return Charset.forName(System.getProperty(FILE_ENCODING));
        } catch (UnsupportedCharsetException e) {
            return Charset.forName("UTF-8");
        }
    }
This is the code of rat-plugin to check whether the file is binary file. I doubt that it should be checked out in UNIx-like system, but on windows, the check is non-binary file, the specific class is in org.apache.rat.document.impl.guesser.BinaryGuesser